django_react_proj/users-fe/src/components/TaskButtonModal/TaskButtonModal.js

```diff
@@ -1,49 +1,34 @@
-import React, { useState, Fragment } from "react";
+import React, {useState} from "react";
 import { Modal, ModalHeader, Button, ModalFooter } from "reactstrap";
-import styles from './TaskButtonModal.module.css';
+import styles from "./TaskButtonModal.module.css";
```

Comment:
This is why you need to have eslint and prettier :D. Created task on boards for those

https://github.com/nokia-wroclaw/innovativeproject-dailyplanner/projects/1#card-58628250
https://github.com/nokia-wroclaw/innovativeproject-dailyplanner/projects/1#card-58628231